aspnetcore/tutorials/first-mvc-app/working-with-sql.md

```diff
@@ -3,7 +3,7 @@ title: Part 5, work with a database in an ASP.NET Core MVC app
 author: rick-anderson
 description: Part 5, add a model to an ASP.NET Core MVC app
 ms.author: riande
-ms.date: 8/16/2019
+ms.date: 11/10/2020
```

Comment:
@wadepickett this is ready

Reply:
...on this.  VS was OK, ran into an issue with VSCode/VS Mac but going back to the beginning to to try again to see if I just missed something.  Will have it soon.